### PR TITLE
ButtonLayout - features, tests and storybook

### PR DESCRIPTION
**Pull request by shlomitc**
Add some sanity tests and preserve original className

### CODE REVIEW

File: src/Button/Button.js [hunk 1]
```diff
@@ -48,10 +48,7 @@ class Button extends WixComponent {
   }
 }
 
-Button.defaultProps = {
-  theme: 'fullblue',
-  height: 'medium'
-};
+Button.defaultProps = Object.assign({}, ButtonLayout.defaultProps);
```
Comment (lbelinsk) [inline src/Button/Button.js]: Button.defaultProps = {...ButtonLayout.defaultProps};

File: stories/ButtonLayout/ExampleControlled.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+class ControlledExample extends Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div style={style}>
+        <ButtonLayout>
+          <div>
```
Comment (lbelinsk) [inline stories/ButtonLayout/ExampleControlled.js]: is the wrapping inner div needed?

File: src/ButtonLayout/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+> General Buttons
+
+## Properties
+
+| propName | propType | defaultValue | isRequired | description |
+|----------|----------|--------------|------------|-------------|
+| theme | string | fullblue | + | The theme of the button, can be transparent, fullred, fullgreen, fullpurple, emptyred, emptygreen, emptybluesecondary, emptyblue, emptypurple, fullblue, transparentblue, whiteblue, whiteblueprimary, whitebluesecondary, close-standard, close-dark, close-transparent, icon-greybackground, icon-standard, icon-standardsecondary, icon-white, icon-whitesecondary
+| height | string | medium | - | The size of the button, can be small, medium or large |
+| active | bool | - | - | - |
```
Comment (lbelinsk) [inline src/ButtonLayout/README.md]: why both active and disabled?

File: stories/ButtonLayout/ExampleControlled.js (hunk 2 above)
Comment (shlomitc) [inline stories/ButtonLayout/ExampleControlled.js]: not needed, will remove it

File: src/ButtonLayout/ButtonLayout.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import styles from './Button.scss';
+import styles from '../Button/Button.scss';
```
Comment (lbelinsk) [inline src/ButtonLayout/ButtonLayout.js]: maybe it makes more sense to put the scss under ButtonLayout. after all it's purpose is layout :)

File: src/ButtonLayout/README.md (hunk 3 above)
Comment (shlomitc) [inline src/ButtonLayout/README.md]: This is what Button already offered as prop

File: src/ButtonLayout/ButtonLayout.js (hunk 4 above)
Comment (shlomitc) [inline src/ButtonLayout/ButtonLayout.js]: right. it was done due to refactor of folders structure

File: src/Button/Button.js (hunk 1 above)
Comment (shlomitc) [inline src/Button/Button.js]: yep

**Pull request merged**